packages/components/src/components/assets/AssetShareIndicator/AssetShareIndicator.tsx

```diff
@@ -10,25 +10,109 @@ const Container = styled.div`
     align-items: center;
     display: flex;
     justify-content: center;
-    width: 48px;
-    height: 48px;
+    width: ${spacingsPx.xxxxl};
+    height: ${spacingsPx.xxxxl};
     border-radius: 50%;
 `;
 
+const rgbToHexColor = (rgbColor: string) => {
+    const [r, g, b] = rgbColor.split('-').map(Number);
+    const hex = [r, g, b].map(x => x.toString(16).padStart(2, '0')).join('');
+
+    return '#' + hex;
+};
+
+const getDominantColor = (imageData: ImageData) => {
```

Comment:
This is a nice pure function that can be separated and unit-tested.

Reply:
Where would you move this function? We can, but IMHO it's not necessary. I don't see any other possible use case for it.

Reply:
Pobably `components/utils/getDominantColor.ts`

The main reason is not re-usability, but to be able to write a test to it (it seems weird to export it from `tsx` component file)

Reply:
It seems that there is `0` :exclamation: tests in components. And by adding a first one, you also "won" the work associated with it (add dependecy, setup it, etc...)

@jvaclavik I thinks its worth it to set it up anyway, but maybe in separate PR?

Reply:
I think it should be pretty easy to setup the testing here.

I prepared PR:
https://github.com/trezor/trezor-suite/pull/13575

Since we will merge it, it should work just with this command: 
`yarn nx run @trezor/components:test`

(and we might need also some new dependency)

Reply:
Ok, I will include setup and the test for `getDominantColor.ts` here. However I'm not sure what is the need of the PR you've opened. We could simply run `yarn test` in `packages/components` if we want to run the component's tests, after I add some basic setup.

Reply:
Done.